var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -165,7 +165,7 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
         # spla removed the openmp option in 1.6.0
         conflicts("^spla@:1.5~openmp", when="+openmp")
 
-    patch("libxc7.patch", when="@7.6.1")
+    patch("libxc7.patch", when="@7.5:7.6.2")
```

Comment:
```suggestion
    patch("libxc7.patch", when="@7.6.0:7.6.1")
```
? Version ranges are inclusive.

Reply:
In case this is a question, yes version ranges are inclusive.